blueprints/ember-cli-nf-graph/index.js

```diff
@@ -0,0 +1,10 @@
+/* globals module */
+module.exports = {
+	normalizeEntityName: function() {},
+
+	afterInstall: function() {
+		return this.addBowerPackagesToProject([
+			{ name: 'd3', target: '~3.5.3' }
```

Comment:
I think we also need to add rxjs, rx-ember:

https://github.com/Netflix/ember-cli-nf-graph/blob/dacef58e71f7ca91bd03cf81d1e36fc92a5a9757/index.js#L11-L13